src/Components/PostInput/PostInput.jsx

```diff
@@ -25,8 +26,27 @@ function PostInput({ children, userObj, setPost, post }) {
 
       <div className="postInput__bottom">
         <div className="postInput__bottom-firstBlock">
-          <GrGallery />
-          <AiOutlineFileGif />
+          <label htmlFor="post image">
```

Comment:
Here, you have given the `htmlFor` attribute but have not given any `id` to the `input`, so either give `id` to the `input` or remove this.

I would suggest giving `id` to the `input`. And I think, words in the `id` should be hyphenated not spaced.

Something like, `post-image`

Reply:
Thanks a lot for reviewing 
I will remove htmlFor then TBH Id is of no use for me so will remove that thanks for suggestion though